### PR TITLE
Add bicep template to create azure resources for benchmarks

### PR DESCRIPTION
**Pull request by JelteF**
This adds two Azure ARM templates that can be used to run a HammerDB
benchmark with very little manual effort.

### CODE REVIEW

File: azure/run-benchmark.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+RESOURCE_GROUP=$1
+shift
+DEPLOYMENT=$1
```
Comment (marcocitus) [inline azure/run-benchmark.sh:5]: would be good to have some basic sanity checks on the arguments

**Comment by marcocitus** [comment]
I got an error:
```
$ ./run-benchmark.sh "$USER-hammerdb-hyperscale" hyperscale
{
  "location": "uksouth",
  "managedBy": null,
  "name": "marco-hammerdb-hyperscale",
  "properties": {
    "provisioningState": "Succeeded"
  },
  "tags": null,
  "type": "Microsoft.Resources/resourceGroups"
}
[Errno 13] Permission denied: '/home/marco/.azure/bin/bicep'
marco@DESKTOP-4OH2JSJ:~/hammerdb-citus2/azure$ ls -l /home/marco/.azure/bin/bicep
-rw-rw-r-- 1 marco marco 0 Dec 21 17:11 /home/marco/.azure/bin/bicep
```

File: azure/driver-vm.bicep [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+__ssh_connection_bashrc__
+
+sudo apt-get install -y postgresql-client-{4}
+git clone https://github.com/citusdata/ch-benchmark.git --branch improvements-jelte
+cd ch-benchmark
+
+{5}
+
+while ! psql -c 'select 1'; do  echo failed; sleep 1; done
+tmux new-session -d -t main -s cloud-init \; send-keys './build-and-run.sh {5}' Enter
```
Comment (marcocitus) [inline azure/driver-vm.bicep]: is it correct that we use {5} here and above as well?

It seemed like the sed commands came after the build-and-run.sh when I tried. This should probably be {6}.

Comment (JelteF) [reply azure/driver-vm.bicep]: yes, I think I forgot to update it when I added the sed commands

Comment (JelteF) [reply azure/driver-vm.bicep]: pushed the  fix now

File: azure/run-benchmark.sh (hunk 1 above)
Comment (marcocitus) [reply azure/run-benchmark.sh:5]: Or fill in the values suggested in the docs by default.

**Review by marcocitus — APPROVED** [review]

**Pull request merged**